packages/core/src/rum/instrumentation/interactionTracking/DdRumUserInteractionTracking.tsx

```diff
@@ -80,14 +82,31 @@ export class DdRumUserInteractionTracking {
         };
 
         try {
-            const jsxRuntime = getJsxRuntime();
-            const originalJsx = jsxRuntime.jsx;
+            const [jsxRuntime, jsxDevRuntime] = getJsxRuntimes();
+            const originalJsx = jsxRuntime?.jsx;
+            const originalDevJsx = jsxDevRuntime?.jsxDEV;
+
             this.originalJsx = originalJsx;
-            jsxRuntime.jsx = (
-                ...args: Parameters<typeof React.createElement>
-            ): ReturnType<typeof React.createElement> => {
-                return this.patchCreateElementFunction(originalJsx, args);
-            };
+            this.originalDevJsx = originalDevJsx;
+
+            if (originalJsx) {
+                jsxRuntime.jsx = (
+                    ...args: Parameters<typeof React.createElement>
+                ): ReturnType<typeof React.createElement> => {
+                    return this.patchCreateElementFunction(originalJsx, args);
+                };
+            }
+
+            if (originalDevJsx) {
```

Comment:
should it be `else if` or both of them can exist at the same time?

Reply:
They should not exist at the same time, but I wanted to be safe in case the implementation changes in different versions of RN.